core/src/main/java/org/mobilitydata/gtfsvalidator/notice/InvalidTimeNotice.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * A field cannot be parsed as time.
+ *
+ * <p>Time must be in the {@code HH:MM:SS} or {@code H:MM:SS} format.
```

Comment:
We also support `HHH:MM:SS` in `GtfsTime`

Reply:
Indeed! We should fix the documentation.

Reply:
@MobilityData/transit-specs To flag this for updating GTFS reference.

Reply:
@aababilov To make the validator internally consistent, I'd recommend updating these Javadocs to what the `GtfsTime` field supports (i.e., add HHH:MM:SS).

Reply:
Done for InvalidTimeNotice.